tools/kagami/src/swarm.rs

```diff
@@ -31,166 +31,461 @@ const FILE_VALIDATOR: &str = "validator.wasm";
 const FILE_CONFIG: &str = "config.json";
 const FILE_GENESIS: &str = "genesis.json";
 const FILE_COMPOSE: &str = "docker-compose.yml";
-const DIR_FORCE_SUGGESTION: &str =
-    "You can pass `--outdir-force` flag to remove the directory without prompting";
+const FORCE_ARG_SUGGESTION: &str =
+    "You can pass `--force` flag to remove the file/directory without prompting";
 const GENESIS_KEYPAIR_SEED: &[u8; 7] = b"genesis";
 
-#[derive(ClapArgs, Debug)]
-pub struct Args {
-    #[command(flatten)]
-    source: ImageSourceArgs,
-    /// How many peers to generate within the docker-compose.
-    #[arg(long, short)]
-    peers: NonZeroU16,
-    /// Target directory where to place generated files.
-    ///
-    /// If the directory is not empty, Kagami will prompt it's re-creation. If the TTY is not
-    /// interactive, Kagami will stop execution with non-zero exit code. In order to re-create
-    /// the directory anyway, pass `--outdir-force` flag.
-    #[arg(long)]
-    outdir: PathBuf,
-    /// Re-create the target directory if it already exists.
-    #[arg(long)]
-    outdir_force: bool,
-    /// Do not create default configuration in the `<outdir>/config` directory.
-    ///
-    /// Default `config.json`, `genesis.json` and `validator.wasm` are generated and put into
-    /// the `<outdir>/config` directory. That directory is specified in the `volumes` field
-    /// of the Docker Compose file.
-    ///
-    /// Setting this flag prevents copying of default configuration files into the output folder. The `config` directory will still be
-    /// created, but the necessary configuration should be put there by the user manually.
-    #[arg(long)]
-    no_default_configuration: bool,
-    /// Might be useful for deterministic key generation.
-    ///
-    /// It could be any string. Its UTF-8 bytes will be used as a seed.
-    #[arg(long, short)]
-    seed: Option<String>,
+mod clap_args {
+    use clap::{Args, Subcommand};
+
+    use super::*;
+
+    #[derive(Args, Debug)]
+    pub struct SwarmArgs {
+        /// How many peers to generate within the Docker Compose setup.
+        #[arg(long, short)]
+        pub peers: NonZeroU16,
+        /// Might be useful for deterministic key generation.
+        ///
+        /// It could be any string. Its UTF-8 bytes will be used as a seed.
```

Comment:
This docstring is used to generate the help message. 

```suggestion
        /// Used for deterministic key-generation. 
        ///
        /// Any valid UTF-8 sequence is acceptable. 
        // TODO: Check for length limitations, and if non-UTF-8 sequences are working. 
```